pkg/cli/registry/login/login.go

```diff
@@ -241,9 +238,12 @@ func (o *LoginOptions) Complete(f kcmdutil.Factory, args []string) error {
 	}
 
 	if len(o.ConfigFile) == 0 {
-		home := homedir.HomeDir()
-		o.ConfigFile = filepath.Join(home, ".docker", "config.json")
-		o.CreateDirectory = true
+		if authFile := os.Getenv("REGISTRY_AUTH_FILE"); authFile != "" {
```

Comment:
- named it in a similar way as REGISTRY_AUTH_FILE
- did not use ORDER in the name because it suggests multiple files (eg separated by comma)
- it could be also called REGISTRY_AUTH_FILE_PREFERENCE.

I suppose it would make sense to have a shared constant for both the REGISTRY_AUTH_FILE and REGISTRY_AUTH_PREFERENCE variables?

Thoughts?